src/cli/cli_tcp.hpp

```diff
@@ -39,6 +39,11 @@
 #include <openthread/tcp.h>
 #include <openthread/tcp_ext.h>
 
+#include <mbedtls/certs.h>
```

Comment:
See https://github.com/Mbed-TLS/mbedtls/blob/development/docs/3.0-migration-guide.md#remove-the-certs-module-from-the-library

Reply:
The include seems to be unneeded.

Reply:
Thanks for letting me know.

Reply:
I've removed it in my next commit.